source/content/guides/support/02-contact-support.md

```diff
@@ -71,12 +69,12 @@ You can download a transcript of your chat with the <Icon icon="arrowDownToLine"
 
 <Alert title="Note" type="info">
 
-Some in-browser ad or tracking blockers can interfere with the **Launch Chat** button. If you don't see it, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has posted more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
+Some in-browser ad or tracking blockers may interfere with the **Start chat** button. If you don't see the button, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has provided more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
 
 </Alert>
 
-## Premium Technical Support and On-Call Services
+## Premium Support Hotline and Emergency Ticket
 
-Diamond and Platinum Account customers can call Pantheon's premium technical support line directly for any technical issues, escalations, site, billing, or overages queries. The phone number can be found in your Workspace, in the Support tab.
+**Diamond** and **Platinum** customers can call the 24/7 Premium Support Hotline for urgent account-related issues. You can find the phone number in the Support tab of your workspace. The phone number can be found in the Support tab of your workspace.
 
-Diamond and Platinum customers also have the option to page Pantheon's operations response team, either via the Pantheon Dashboard or via an emergency 800-number. Pantheon on-call immediately escalates to the on-call engineering team. The scope of on-call support is limited to emergencies and business critical issues.
+**Diamond** and **Platinum** customers also have the option to open an emergency ticket for business-critical issues. Pantheon escalates emergency tickets to our engineering team, even during off-hours. If your issue is non-business-critical, please open a general support ticket instead.
```

Comment:
Avoid confusion between "engineering" and "support" teams for tickets. It's also a good idea to outline where this action can be performed. 

Suggestion:
> **Diamond** and **Platinum** customers also have the option to open an emergency ticket for business-critical issues. Pantheon escalates emergency tickets to our support team, even during off-hours. Emergency tickets can be created from the Support page of either the [Workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces) or [Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) of the affected site. If your issue is non-business-critical, please open a general support ticket instead.

Reply:
I think it would be beneficial to include the term 'engineering' / "support engineers" to emphasize that the emergency ticket is specifically intended for technical issues.

Reply:
we have used "support engineers" in previous iterations I believe, so maybe we stick with that for now